pkg/clusterresourceoverride/mutator_test.go

```diff
@@ -412,6 +413,139 @@ func TestMutator_OverrideCPULimit(t *testing.T) {
 	}
 }
 
+func TestMutator_OverrideSelinuxFix(t *testing.T) {
+	tests := []struct {
+		name    string
+		mutator func() *podMutator
+		input   *corev1.Pod
+		assert  func(t *testing.T, resources *corev1.Pod)
+	}{
+		{
+			name: "Selinux Enabled with PVC and label",
+			mutator: func() *podMutator {
+				return &podMutator{
+					config: &Config{
+						EnableSelinuxFix:          true,
+						LimitCPUToMemoryRatio:     100,
+						CpuRequestToLimitRatio:    100,
+						MemoryRequestToLimitRatio: 100,
+					},
+				}
+			},
+			input: &corev1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"selinuxrelabelfix.admission.node.openshift.io/enabled": "true",
+					},
+				},
+				Spec: corev1.PodSpec{
+					Volumes: []corev1.Volume{
+						{
+							Name: "test-pv-storage",
+							VolumeSource: corev1.VolumeSource{
+								PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+									ClaimName: "test-pvc",
+								},
+							},
+						},
+					},
+					Containers: []corev1.Container{
+						{
+							Name:  "container_1",
+							Image: "busybox",
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "test-pvc",
+									MountPath: "/data",
+								},
+							},
+						},
+					},
+				},
+			},
+			assert: func(t *testing.T, pod *corev1.Pod) {
+				require.Equal(t, pod.Spec.SecurityContext.SELinuxOptions.Type, SpcType)
+			},
+		},
+		{
+			name: "Selinux Enabled with PVC without label",
+			mutator: func() *podMutator {
+				return &podMutator{
+					config: &Config{
+						EnableSelinuxFix:          true,
+						LimitCPUToMemoryRatio:     100,
+						CpuRequestToLimitRatio:    100,
+						MemoryRequestToLimitRatio: 100,
+					},
+				}
+			},
+			input: &corev1.Pod{
+				Spec: corev1.PodSpec{
+					Volumes: []corev1.Volume{
+						{
+							Name: "test-pv-storage",
+							VolumeSource: corev1.VolumeSource{
+								PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
+									ClaimName: "test-pvc",
+								},
+							},
+						},
+					},
+					Containers: []corev1.Container{
+						{
+							Name:  "container_1",
+							Image: "busybox",
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "test-pvc",
+									MountPath: "/data",
+								},
+							},
+						},
+					},
+				},
+			},
+			assert: func(t *testing.T, pod *corev1.Pod) {
+				require.Nil(t, pod.Spec.SecurityContext)
+			},
+		},
+		{
+			name: "Selinux Enabled without PVC with label",
```

Comment:
Does it make sense to add a test like `Selinux Enabled without PVC with label and existing SecurityContext` to check that the proposed logic does not change the existing data when no PersistentVolumeClaim is found?

Reply:
Added that test.